app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/importlogins/ImportLoginsScreen.kt

```diff
@@ -0,0 +1,253 @@
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.x8bit.bitwarden.R
+import com.x8bit.bitwarden.data.platform.annotation.OmitFromCoverage
+import com.x8bit.bitwarden.ui.platform.base.util.EventsEffect
+import com.x8bit.bitwarden.ui.platform.base.util.standardHorizontalMargin
+import com.x8bit.bitwarden.ui.platform.components.appbar.BitwardenTopAppBar
+import com.x8bit.bitwarden.ui.platform.components.appbar.NavigationIcon
+import com.x8bit.bitwarden.ui.platform.components.button.BitwardenFilledButton
+import com.x8bit.bitwarden.ui.platform.components.button.BitwardenOutlinedButton
+import com.x8bit.bitwarden.ui.platform.components.dialog.BitwardenTwoButtonDialog
+import com.x8bit.bitwarden.ui.platform.components.scaffold.BitwardenScaffold
+import com.x8bit.bitwarden.ui.platform.components.util.rememberVectorPainter
+import com.x8bit.bitwarden.ui.platform.theme.BitwardenTheme
+import com.x8bit.bitwarden.ui.vault.feature.importlogins.handlers.ImportLoginHandler
+import com.x8bit.bitwarden.ui.vault.feature.importlogins.handlers.rememberImportLoginHandler
+
+/**
+ * Top level component for the import logins screen.
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun ImportLoginsScreen(
+    onNavigateBack: () -> Unit,
+    viewModel: ImportLoginsViewModel = hiltViewModel(),
+) {
+    val state by viewModel.stateFlow.collectAsStateWithLifecycle()
+    val handler = rememberImportLoginHandler(viewModel = viewModel)
+
+    EventsEffect(viewModel = viewModel) { event ->
+        when (event) {
+            ImportLoginsEvent.NavigateBack -> onNavigateBack()
+        }
+    }
+
+    ImportLoginsDialogContent(state = state, handler = handler)
+
+    val scrollBehavior = TopAppBarDefaults.pinnedScrollBehavior(rememberTopAppBarState())
+    BitwardenScaffold(
+        modifier = Modifier
+            .fillMaxSize()
+            .nestedScroll(scrollBehavior.nestedScrollConnection),
+        topBar = {
+            BitwardenTopAppBar(
+                title = stringResource(R.string.import_logins),
+                navigationIcon = NavigationIcon(
+                    navigationIcon = rememberVectorPainter(R.drawable.ic_close),
+                    onNavigationIconClick = handler.onCloseClick,
+                    navigationIconContentDescription = stringResource(R.string.close),
+                ),
+                scrollBehavior = scrollBehavior,
+            )
+        },
+    ) { innerPadding ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(paddingValues = innerPadding),
```

Comment:
Does this column do anything?

Reply:
Yeah, so thought process here is that this screen is actually going to support the rest of the "instruction steps inside this Scaffold with `Crossfade` this is what they are doing on iOS after talking with design, so the purpose of this will be to give the Scaffold padding for free to those.

Reply:
Discussed offline a little further, the `Column` will become a `Crossfade` in the next PR.

Reply:
Discussed offline, will make change to this to be the `CrossFade` function and remove the `Column`